deployment/deployment.go

```diff
@@ -372,6 +371,22 @@ func getDeployment(deploymentId string, client *houston.Client) (*houston.Deploy
 	return &r.Data.GetDeployment, nil
 }
 
+func getUpdateDeploymentAirflow(deploymentId string, client *houston.Client) (*houston.Deployment, error) {
```

Comment:
whats the difference between this function and getDeployment?

Reply:
Nothing :P 

I took a closer look and now understand the issue was actually mocked API data missing from the tests. I removed the unnecessary extra stuff that I added. This PR should be a bit more straightforward now.